TCAT/Controllers/InformationViewController.swift

```diff
@@ -42,138 +43,137 @@ class InformationViewController: UIViewController, UITableViewDataSource, UITabl
+        tableView.delegate = self
+        tableView.backgroundColor = Colors.backgroundWash
+        tableView.separatorColor = Colors.dividerTextField
+        tableView.showsVerticalScrollIndicator = false
+
+        setupHeaderView()
+        tableView.tableHeaderView = headerView
+
         view.addSubview(tableView)
+    }
+
+    func setupHeaderView() {
+        headerView = UIView(frame: CGRect(x: 0, y: 0, width: headerWidth, height: headerHeight))
+
+        setupDismissButton()
+        setupHiddenLabel()
+        setupTcatImage()
+        setupTitleLabel()
+        setupDescriptionLabel()
+
+        setupHeaderViewConstraints()
+    }
 
+    func setupDismissButton() {
         dismissButton.addTarget(self, action: #selector(dismissTapped), for: .touchUpInside)
         dismissButton.setTitle(Constants.Buttons.done, for: .normal)
         navigationItem.rightBarButtonItem?.setTitleTextAttributes(
             CustomNavigationController.buttonTitleTextAttributes, for: .normal
         )
         let backButtonItem = UIBarButtonItem(customView: dismissButton)
         navigationItem.setRightBarButton(backButtonItem, animated: false)
+    }
 
+    func setupHiddenLabel() {
         hiddenLabel.font = .getFont(.regular, size: 16)
         hiddenLabel.textColor = Colors.primaryText
         hiddenLabel.text = Constants.InformationView.magicSchoolBus
         hiddenLabel.textAlignment = .center
         hiddenLabel.backgroundColor = .clear
-        hiddenLabel.snp.makeConstraints { (make) in
-            make.top.equalTo(view.safeAreaLayoutGuide).offset(86)
-            make.centerX.equalToSuperview()
-            make.height.equalTo(20)
-        }
+        headerView.addSubview(hiddenLabel)
+    }
 
+    func setupTcatImage() {
         tcatImage.image = UIImage(named: "tcat")
         tcatImage.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(busTapped)))
         tcatImage.isUserInteractionEnabled = true
-        tcatImage.snp.makeConstraints { (make) in
-            make.centerX.equalToSuperview()
-            make.top.equalTo(view.safeAreaLayoutGuide).offset(44)
-            make.width.equalTo(tcatImage.snp.height).multipliedBy(2.5)
-            make.leading.equalToSuperview().offset(40)
-            make.trailing.equalToSuperview().offset(-40)
-        }
+        headerView.addSubview(tcatImage)
+    }
 
+    func setupTitleLabel() {
         titleLabel.font = .getFont(.medium, size: 16)
         titleLabel.textColor = Colors.primaryText
         titleLabel.text = Constants.InformationView.madeBy
         titleLabel.backgroundColor = .clear
-        titleLabel.snp.makeConstraints { (make) in
-            make.top.equalTo(tcatImage.snp.bottom).offset(44)
-            make.centerX.equalToSuperview()
-            make.bottom.equalTo(descriptionLabel.snp.top).offset(-12)
-            make.height.equalTo(20)
-        }
+        headerView.addSubview(titleLabel)
+    }
 
+    func setupDescriptionLabel() {
```

Comment:
nitpick, these functions should be alphabetized, which means that this should go above the others

Reply:
i actually think it should be ordered in the order that they appear in the screen since thats how we build screens. alphabetizing it doesn't bring that much benefit especially since XCode has jump to definition